<!DOCTYPE HTML>
<!--
	Strata by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Grayson Chadwick's Research Works</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="images/icon.png">
	</head>
	<body class="is-preload">

		<!-- Header -->
			<header id="header">
				<div class="inner">
					<a href="#" class="image avatar"><img src="images/avatar.png" alt="" /></a>
					<h1><strong>Grayson L Chadwick, PhD</strong></h1><h1>Microbial Physiologist, Geneticist, Ecologist</h1><h1>Bioinformatician</h1>
				</div>
			</header>

		<!-- Main -->
			<div id="main">
				<!-- One -->
					<section id="one">
						<header class="major">
							<h2>Unusual Microbiology</h2>
						</header>
						
						<p>I am a scientist inspired by the diversity of microbial life. My research covers a range of topics including microbial physiology, ecology, evolution, symbiosis, and biogeochemistry. Focuses include: archaea involved in the global methane cycle, phototrophic sulfur oxidizers with unusual communal behaviors, bacteria that grow on electrodes, hyperthermophiles thriving near the boiling point of water, and extreme alkaliphiles adapted to life at pH 12. Many of these organisms are not available in pure culture, so my work has involved field campaigns to the deep ocean and terrestrial springs with unusual geochemistries; along with the application of a range of techniques developed for uncultured organisms. I combine these environmental microbiology approaches with mathematical modeling and traditional biochemical and genetic techniques to learn new and interesting biology. Examples of questions that have driven my research are:</p>
						
						<ul>
						<li>What evolutionary steps led to the transition between methane-producing and methane-oxidizing physiologies in the Archaea?</li>
						<li>How do different types of syntrophic interaction determine the physical organization of cell types in symbiotic associations?</li>
						<li>What role does nitrogen fixation play in marine methane seep ecosystems?</li> 
						<li>How does extracellular electron transport work and what are the spatial limitations of this process?</li>
						<li>How can sufficiently large and fast bacteria escape low-Reynolds number, diffusion-dominated regimes?</li>
						</ul>
						
						<p>Below I have highlighted how some of my work has contributed to answering these questions, summarizing the main takeaways from studies that are thematically related that I have led or contributed to as a co-author.</p>

						<p>The <a href="#recent">Recent Projects</a> section is mostly completed work from my PhD in the lab of Victoria Orphan at Caltech and the <a href="#current">Current Projects</a> section describes ongoing postdoctoral work in the lab of Dipti Nayak at UC Berkeley. My prior work focused more heavily on environmental microbiology questions and techniques, while the goal of my current work is to gain experience with genetic manipulation and protein biochemistry on organisms in pure culture. In the future, I hope to carry out research that requires both approaches at the interface between fundamental discovery and rigorous experimentation.</p>

						<p>I have had the great pleasure of collaborating with a broad group of fantastic people on these projects. The best experiences I have had in science are when a unique technique or expertise I can bring to a colleague's project helps improve their story; while many of the successes of projects I have led are due to finding a collaborator with the right expertise that I lack. These interactions have left me with a broader appreciation for and understanding of biology. My ideal future research program will involve working with a wide variety of collaborations across many different disciplines.</p>

						<p><a href="Chadwick_CV_2022_07.pdf">Curriculum Vitae</a><br>
				         <a href="https://scholar.google.com/citations?hl=en&user=XaywC5MAAAAJ">Google Scholar</a></p>
					</section>
				<!-- Three -->
					<section id="three">
						<h2 id="current">Current Projects</h2>
						<div class="row">
							<article class="col-4 col-12-xsmall work-item">
							  <a href="images/fulls/09.png" class="image fit thumb"><img src="images/thumbs/09.png" alt="" /></a>
								<h3>Understanding the assembly of the methane generating enzyme in <em>Methanosarcina acetivorans</em></h3>
							  <p>Nearly all biologically produced methane comes from Archaea known as methanogens. Whether it is emitted from a cow, natural wetlands, rice paddies or bioreactors, methane in these ecosystems is produced almost entirely from methanogenic Archaea living in oxygen-free niches. The enzyme responsible for methane production is methyl-coenzyme M reductase (Mcr), which contains 6 protein subunits, unique cofactors and a raft of post-translational modifications. In this project I am using Cas9-based genome editing, physiology and structural biology to understand what intermediate states exist in Mcr assembly. Genetic modification of the Mcr operon in the native host has allowed for the purification of Mcr assembly intermediates in large enough quantities for our collaborator Aaron Joiner to generate high resolution models from single-particle cryoEM. Our ongoing work on this project focuses on refining these models and attempting to generate a mechanistic understanding of the Mcr-associated proteins that help with the assembly process.</p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/10.png" class="image fit thumb"><img src="images/thumbs/10.png" alt="" /></a>
								<h3>Physiological and transcriptional response to methane inhibition in a methanogenic archaea</h3>
								<p>The methane generating enzyme (Mcr) is required for the viability of methanogenic Archaea because  all methanogens must make methane to survive. Mcr is also found in the methanotrophic archaea I studied in my PhD, where it is used in the reverse direction to consume methane. In both cases Mcr is one of the most abundant proteins in the cell, and the mRNA encoding it is one of the most prevalent in the transcriptome. This has led to the common assumption that Mcr is rate-limiting in both metabolisms. In this project I have placed the Mcr operon under the control of an inducible promoter allowing for tight control the amount of Mcr that is expressed. We are interested in understanding the aberrant (linear) growth phenotypes we observe when Mcr transcription is inhibited as well as understanding the global transcriptional response to Mcr limitation in <i>Methanosarcina acetivorans.</i> We hope to test whether Mcr abundance is actually limiting methanogenesis under ideal lab conditions, and see whether our induced limitation reveals unknown backup metabolic processes.</p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/11.png" class="image fit thumb"><img src="images/thumbs/11.png" alt="" /></a>
								<h3>Developing the <em>Archaeoglobi</em> into a model system for the study of methane metabolism in Archaea </h3>
								<p>The past few years have seen tectonic shifts in our understanding of how broadly distributed methane metabolism is in the Archaea. In a 2015 study we found the first metagenomic evidence for the methane generating enzyme (Mcr) outside of the Euryarchaeota—the clade of Archaea previously thought to contain all methane metabolizing archaea (<a href="#4">4</a>). Subsequent studies by many groups have validated and extended these observations to an ever-increasing number of uncultivated Archaea. A study I helped with in 2019 found Mcr homologs in an uncultivated member of the hyperthermophilic archaeal class <em>Archaeoglobi</em> (<a href="#12">12</a>). A handful of the <em>Archaeoglobi</em> have been isolated in pure culture, including the sulfate reducing <em>Archaeoglobus sp.</em> and the metal and nitrate reducing <em>Ferroglobus</em> and <em>Geoglobus</em>; all of which can use a broad range of complex carbon compounds. My hope for this project is to develop the <em>Archaeoglobi</em> into genetically tractable model organisms to study how methanogenesis and methane oxidation can be gained and lost in Archaea. An added benefit of studying these organisms is that one species contains and extended complex I operon, which is the focus of <a href="#complexi">one of my past research projects</a>.</p>
							</article>
						</div>
				</section>
				<section id="two">
						<h2 id="recent">Recent Projects</h2>
						<div class="row">
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/01.png" class="image fit thumb"><img src="images/thumbs/01.png" alt="" /></a>
								<h3>Evolutionary and Phylogenetic Analysis of Anaerobic Methanotrophic Archaea</h3>
								<p>This project led by myself and co-first author <a href="https://ctskennerton.github.io/">Connor Skennerton</a> was a multi-institute collaborative analysis of all currently available metagenome-assembled genomes (MAGs) of Anaerobic Methanotrophic Archaea (ANME) (<a href="#27">27</a>). This work represents the state-of-the-art in our understanding of how the symbiosis between ANME and their bacterial partners functions to remove this potent greenhouse gas from marine methane seeps. The "parts list" of ANME-specific proteins we highlight here are thought to be crucial for the adaptation of a methanogenic ancestor to a methanotrophic lifestyle. The most exciting of these parts was the acquisition of extracellular electron transport machinery, which allows the ANME to share electrons directly with their syntrophic partner without a diffusible intermediate. </p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/06.png" class="image fit thumb"><img src="images/thumbs/06.png" alt="" /></a>
								<h3>Characterization of an ultrafast swimming phototrophic bacteria</h3>
								<p>While working as a teaching assistant at the Woods Hole Microbial Diversity course I enriched a new genus of purple sulfur bacteria based on their communal swimming and biofilm forming behavior (<a href="#21">21</a>). Nearly all bacteria are small enough and slow enough to live in what is called a low-Reynolds number regime, where substrate supply is determined entirely by diffusive and not advective transport. High speed video microscopy and single particle tracking revealed this new bacteria to be the fastest swimming phototrophic bacteria, with top speeds of just over 800 microns per second. This size and speed puts them in an intermediate regime where our modelling suggests the fluid flow they produce can actually increase their substrate supply beyond what diffusion alone can generate.</p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/07.png" class="image fit thumb"><img src="images/thumbs/07.png" alt="" /></a>
								<h3>Mapping the activity of individual cells in symbiotic methane oxidizing consortia</h3>
								<p>The anaerobic oxidation of methane is a hugely important process in many marine and terrestrial environments that prevents methane, a strong greenhouse gas, from escaping into the atmosphere. The microorganisms responsible for this are archaea known as ANME that live in symbiotic consortia with sulfate reducing bacteria (SRB). In this project led by myself and co-first author <a href="https://sites.google.com/elsi.jp/mcglynn/">Shawn McGlynn</a> we combined stable isotope incubation experiments coupled to fluorescent in situ hybridization (FISH) and nanometer-scale secondary ion mass spectrometry (nanoSIMS) to map the activity of individual cells in relation to other cells in their structured community (<a href="#5">5</a>). These spatially-indexed activity measurements, combined with genomic analysis and modelling, indicate that the ANME-SRB syntrophy is based on the direct transfer of electrons between the symbiotic partners.</p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/03.png" class="image fit thumb"><img src="images/thumbs/03.png" alt="" /></a>
								<h3>Mapping the activity of electroactive microbes at sub-micron resolution</h3>
								<p>Electroactive microorganisms such as <em>Geobacter sulfurreducens</em> are able to grow using anodes as their terminal electron acceptor. These features have resulted in great interest in using these organisms in bioelectrochemical reactors, where organic matter degradation is linked directly to electricity generation. The current density generated in these systems is an important value that determines the applicability of these technologies, so understanding how cell growth is limited in these communities is of great importance. In this work led by myself and co-first author <a href="https://scholar.google.com/citations?user=j_lVH9gAAAAJ&hl=en">Fernanda Jiménez Otero</a> we used stable isotope pulses and nanoSIMS imaging to map the activity of <em>G. sulfurreducens</em> at sub-micron resolution within anodic biofilms. We showed that growth largely decreases with distance from the anode surface, suggesting that biofilm conductivity, not carbon substrate supply, was limiting in our reactors (<a href="#15">15</a>). We further showed how changes in the applied potential on the anode can effect the pattern of cell activity in the biofilms. </p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/14.png" class="image fit thumb"><img src="images/thumbs/04.png" alt="" /></a>
								<h3>NanoSIMS and modeling shed light on extracellular electron transfer mechanisms</h3>
								<p>In a series of three papers led by <a href="https://scholar.google.com/citations?user=_E2RHlQAAAAJ&hl=en">Xiaojia He</a> during his time at the University of Georgia we combined our FISH-nanoSIMS approach to visualize the metabolic activity of single cells in complex microbial communities with Xiaojia's metabolic modeling approaches. Through this series of papers we were able to iterate between experimentation and modeling to produce a detailed understanding of the interplay between physical community structure and overall function of the community. These papers have greatly advanced our understanding of what is possible in the ANME-SRB symbiosis, as well as helped shed light on what limits the current densities that can be achieved by Geobacter biofilms grown on electrodes (<a href="#18">18</a>, <a href="#13">13</a>, <a href="#17">17</a>). Further application of the isotope probing and imagine techniques, in conjunction with genetic manipulation and bioelectrochemical analysis, helped explain increased current density in <em>Geobacter</em> mutants led by <a href="https://scholar.google.com/citations?user=j_lVH9gAAAAJ&hl=en">Fernanda Jiménez Otero</a> (<a href="#20">20</a>).</p>
							</article>
							<article id="complexi" class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/05.png" class="image fit thumb"><img src="images/thumbs/05.png" alt="" /></a>
								<h3>Evolutionary convergence on structural modifications to respiratory Complex I for increased proton pumping</h3>
								<p>In a comprehensive phylogenomic analysis of respiratory Complex I operons across all prokaryotic genomes I found a repeated pattern of incorporation of an additional proton pumping subunit. Leveraging recent crystal structures of Complex I the acquisition of these pumping subunits co-occurs with amino acid insertions necessary for changing the quaternary structure to incorporate the extra subunit. Remarkably, this genetic and structural reorganization evolved independently in three unrelated phylogenetic groups. These observations help explain a longstanding mystery about carbon fixation in ammonium-oxidizing Nitrospira (<a href="#10">10</a>). In a comparative genomic study led by <a href="https://scholar.google.com/citations?user=3no8Uy8AAAAJ&hl=en&oi=ao">Hank Yu</a> on the newly-discovered chemolithoautotrophic manganese oxidizing bacteria we found that these organisms have iterated on this evolutionary process, bringing in yet another pumping subunit, indicating the ease with which this design principle can be repeatedly utilized for augmenting microbial energy metabolism (<a href="#24">24</a>).</p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
							  <a href="images/fulls/08.png" class="image fit thumb"><img src="images/thumbs/08.png" alt="" /></a>
								<h3>Investigations into hyperalkaliphilic life in serpentinizing systems</h3>
								<p>Serpentinization is a geologic process that can produce marine hydrothermal vents or terrestrial springs that are characterized by very low redox potential and very high pH. The chemical gradients that occur when these fluids mix with more oxidizing waters of the ocean or the oxygenated atmosphere make for a great habitat for extremophilic microorganisms. In a project led by Leah Trutschel in <a href="www.electromicrobiology.org">Annie Rowe</a>'s lab, I helped with a series of field campaigns to characterize the chemistry and microbiology of Ney Springs, a serpentinizing spring from Northern California with the most extreme eH/pH of any natural waters on Earth (<a href="#26">26</a>). Ongoing work on this system is attempting cultivation of some of the dominant organisms in the pH 12 waters, as well as continued monitoring of the microbial community dynamics over time.</p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/13.png" class="image fit thumb"><img src="images/thumbs/13.png" alt="" /></a>
								<h3>Understanding the breadth and significance of nitrogen fixation in marine methane seep ecosystems</h3>
								<p>During my time in the Orphan lab I worked on three studies all led by <a href="https://dekaslab.stanford.edu/">Anne Dekas</a> that helped us better understand the role of nitrogen fixation in marine sediments. In this work I helped set up stable isotope probing incubations with 15N-labeled nitrogen gas so that we could trace bulk nitrogen fixation rates with Elemental Analyzer Isotope Ratio Mass Spectrometry, as well as trace nitrogen fixation into individual organisms using our combined FISH-nanoSIMS approach. I also extracted DNA and RNA from our sediment incubations to help understand which organisms in the community contained the genetic potential for nitrogen fixation and who were actively transcribing nitrogenase genes. Most of our work focused on methane seep ecosystems(<a href="#2">2</a>,<a href="#6">6</a>), but we also compared the nitrogen fixation rates and active organisms between these ecosystems and other marine sediments such as background non-seep sediments and the communities surrounding whalefalls(<a href="#9">9</a>).</p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/15.png" class="image fit thumb"><img src="images/thumbs/15.png" alt="" /></a>
								<h3>NanoSIMS investigations of other structured microbial communities</h3>
								<p>I was lucky enough to be a collaborator on a number of other projects where we could use our FISH-nanoSIMS approach to help other groups answer interesting questions of their structure microbial communities. In a project led by <a href="https://scholar.google.com/citations?user=Zwux6MQAAAAJ">DeAnna Bublitz</a> I helped design experiments and develop a sample preparation strategy to trace isotopically labelled D-alanine into the peptidoglycan layer of <em>Moranella sp.</em> which live inside <em>Tremblaya sp.</em>, which in turn live inside of special bacteriocyte cells in Mealybugs (<a href="#14">14</a>). This application of phylogenetic imaging and isotope imaging helped us make the case that peptidoglycan production of the innermost bacterial symbiont was dependent on biosynthetic processes that have been exported to the insect genome. In another study led by <a href="https://scholar.google.com/citations?user=rZ58X6QAAAAJ&hl=en&oi=ao">Julia Schwartzman</a> we used stable isotope probing to understand the nutrient consumption dynamics of vibrio cultures that form bizarre spherical biofilms, where phenotypic differentiation occurs, resulting in an outer layer of immobile cells forming a shell around a rapidly swimming motile inner core (<a href="#25">25</a>).</p>
							</article>
						</div>
					</section>
				<!-- Four -->
			  		<section id="four">
						<h2>Mentorship and Outreach</h2>
						<div class="row">
							<article class="col-4 col-12-xsmall work-item">
							  <a href="images/fulls/17.png" class="image fit thumb"><img src="images/thumbs/17.png" alt="" /></a>
								<h3>Remote metorship during COVID-19</h3>
							  <p>During the COVID-19 pandemic many summer research programs were forced to be remote, which is particularly hard in the biological sciences where many of the students hope to get laboratory experience. During the summer of 2020 I volunteered as a mentor for the Research Science Institute (RSI) at MIT, a program I participated in as a student in 2006. During that project my summer student and I expanded on the complex I analysis I did in my PhD (<a href="#10">10</a>), generating a series of python programs to screen the entire GTDB prokaryote genome database for complex I operons and layer these gene clusters on phylogentic trees to easily visualize gene synteny changes through evolutionary time. In the summer of 2021 I mentored two additional RSI students and two students from NIH's Bridges to Baccalaureate (B2B) Research Training Program run by the Molecular and Cell Biology department at UC Berkeley. Projects included evolution of flu virus spike proteins and the incorporation of the non-standard amino acid pyrrolysine into proteins in methanogenic archaea.</p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/12.png" class="image fit thumb"><img src="images/thumbs/12.png" alt="" /></a>
								<h3>In-person mentorship</h3>
								<p>In 2018 I served as a summer research mentor for the Amgen Scholars Program. My mentee and I developed protocols for coupling fluorescence in situ hybridization with laser capture microdissection of individual ANME-SRB aggregates from methane seep sediments. Individual captured aggregates were subjected to DNA extraction and PCR for marker genes to link the spatial organization of the community with strain-level information on the partner organisms.<br>
								Summer 2022 I am following up with one of my remote B2B students from last summer who wanted to come back and get some laboratory experience. Together we are working on making some of the genome editing contructs and testing transformation protocols in my <em>Archaeoglobus</em> genetics project.</p>
							</article>
							<article class="col-4 col-12-xsmall work-item">
								<a href="images/fulls/16.jpg" class="image fit thumb"><img src="images/thumbs/16.png" alt="" /></a>
								<h3>Science outreach at El Cerrito High School </h3>
								<p>Starting in 2020 the Miller Insitute at UC Berkeley began working with science teachers at El Cerrito High School to develop outreach aimed at inspiring and educating students about careers in science. I participated virtually during pandemic school closures, and then took over organizing these outreach efforts in early 2021, transitioning them to in-person format. In these classroom visits I share some of my favorite science demos and answer questions about scientific careers. For biology I have brought microscopes and a variety of everyday examples of symbiosis and parasatism in lichens and dodder. For the physics classes I bring a cloud chamber for visualizing cosmic rays and ferrofluid for to visualize magnetic field lines.</p>
							</article>
						</div>
					</section>
				<!-- Five -->
			  		<section id="five">
						<div class="row">
							<article class="col-4 col-12-xsmall work-item">
								<h2>Awards</h2>
								<p>• 2020 – 2023 Miller Research Fellowship<br>
									• 2017 – 2019 NIH Predoctoral Training Grant in Molecular and Cell Biology<br>
									• 2016 Gordon P. and Virginia G. Eaton Graduate Fellowship<br>
									• 2015 Caltech CEMI Fellowship<br>
									• 2013 Rose Hills Fellowship<br>
									• 2010 NCAA Men’s Soccer Captain, First-Team All-Conference, Brine Award of Distinction<br> 
									• 2009, 2010 Caltech Upperclass Merit Scholarship<br>
									• 2009 NCAA Men’s Soccer Captain, Second-Team All-Conference, Brine Award of Distinction<br>
									• 2009 Amgen Scholars Program<br>
									• 2009 George W. and Bernice E. Green Memorial Prize<br>
									• 2008, 2010 Caltech Summer Undergraduate Research Fellowship<br>
									• 2008 Richard G. Brewer Prize in Physics</p>
							</article>
							<article class="col-2 col-12-xsmall work-item">
								<h2>Metrics</h2>
								<p>As of July 2022<br>
									• Publications 26<br>
									• Citations 1933<br>
									• h-index 14<br>
									• i10-index 15<br>
								</p>
							</article>
							<article class="col-6 col-12-xsmall work-item">
								<h2>Teaching and Service</h2>
								<p>
								• 2022 Co-chair Gordon Research Seminar Microbial One-Carbon Conversion from the Microscale to the Global Scale<br><br>
								• 2010 – 2019 Teaching assistant for Caltech courses: Microbial Ecology, Evolution, Microbial Metabolic Diversity, Bioengineering Bootcamp, The Great Ideas of Biology, Principles of Biology<br><br>
								• 2017 - 2019, 2022 Teaching assistant for International Geobiology Course, Caltech<br><br>   • 2016 Teaching assistant for Microbial Diversity Course, Marine Biological Laboratory at Woods Hole<br><br>
								• 2016, 2019 Organizer Southern California Geobiology Symposium<br><br>
								• 2017 – 2018 Organizer Caltech Geoclub Seminar Series<br><br>
								• Reviewer for: Nature Microbiology, Science, Frontiers in Microbiology, Environmental Microbiology, Communications Biology, mSystems, Limnology and Oceanography: Methods, Elementa, Biogeosciences, NASA Astrobiology External Reviewer

								</p>
							</article>
						
						</div>
					</section>
				<!-- Six -->
					<section id="six">
						<h2>Publications</h2>
						<p id="27">27: <strong><u>Chadwick, G. L.</u></strong>*, Skennerton, C. T.*, Laso-Pérez, R., Leu, A. O., Speth, D. R., Yu, H., ... & Orphan, V. J. (2022). <a href="https://doi.org/10.1371/journal.pbio.3001508">Comparative genomics reveals electron transfer and syntrophic mechanisms differentiating methanotrophic and methanogenic archaea.</a> PLoS biology, 20(1), e3001508. *Authors contributed equally to this work. </p>
							<p id="26">26: Trutschel, L. R., <strong><u>Chadwick, G. L.</u></strong>, Kruger, B., Blank, J. G., Brazelton, W. J., Dart, E. R., & Rowe, A. R. (2022). <a href="https://doi.org/10.1016/j.scitotenv.2022.155492">Investigation of microbial metabolisms in an extremely high pH marine-like terrestrial serpentinizing system: Ney Springs.</a> Science of The Total Environment, 836, 155492. </p>
							<p id="25">25: Schwartzman, J. A., Ebrahimi, A., <strong><u>Chadwick, G. L.</u></strong>, Sato, Y., Roller, B. R., Orphan, V. J., & Cordero, O. X. (2022). <a href="https://doi.org/10.1016/j.cub.2022.06.011">Bacterial growth in multicellular aggregates leads to the emergence of complex life cycles.</a> Current Biology. </p>
							<p id="24">24: Yu, H., <strong><u>Chadwick, G. L.</u></strong>, Lingappa, U. F., & Leadbetter, J. R. (2022). <a href="https://doi.org/10.1128/mbio.03421-21">Comparative Genomics on Cultivated and Uncultivated Freshwater and Marine “Candidatus Manganitrophaceae” Species Implies Their Worldwide Reach in Manganese Chemolithoautotrophy.</a> mBio, 13(2), e03421-21. </p>
							<p id="23">23: Yu, H., Skennerton, C. T., <strong><u>Chadwick, G. L.</u></strong>, Leu, A. O., Aoki, M., Tyson, G. W., & Orphan, V. J. (2022). <a href="https://doi.org/10.1038/s41396-021-01047-0">Sulfate differentially stimulates but is not respired by diverse anaerobic methanotrophic archaea.</a> The ISME Journal, 16(1), 168-177. </p>
							<p id="22">22: Brazelton, W. J., McGonigle, J. M., Motamedi, S., Pendleton, H. L., Twing, K. I., Miller, B. C., Lowe, W. J., Hoffman, A. M., Prator, C. A., <strong><u>Chadwick, G. L.</u></strong> ... & Lang, S. Q., (2022). <a href="https://doi.org/10.1101/2022.01.25.477282">Metabolic strategies shared by basement residents of the Lost City hydrothermal field.</a> bioRxiv. Note: in final revision at Applied and Environmental Microbiology. </p>
							<p id="21">21: <strong><u>Chadwick, G. L.</u></strong> (2021). <a href="https://thesis.library.caltech.edu/13842/37/Chadwick_Thesis_v3.pdf">How to Beat Diffusion: Explorations of Energetics and Spatial Relationships in Microbial Ecosystems</a> (Doctoral dissertation, Caltech). </p>
							<p id="20">20: Jiménez Otero, F., <strong><u>Chadwick, G. L.</u></strong>, ... & Bond, D. R. (2021). <a href="https://doi.org/10.1128/AEM.00706-21">Evidence of a streamlined extracellular electron transfer pathway from biofilm structure, metabolic stratification, and long-range electron transfer parameters.</a> Applied and Environmental Microbiology, 87(17), e00706-21. </p>
							<p id="19">19: Lingappa, U.F., Yeager, C.M., Sharma, A., Lanza, N.L., Morales, D.P., Xie, G., Atencio, A.D., <strong><u>Chadwick, G. L.</u></strong>, ... & Fischer, W. W. (2021). <a href="https://doi.org/10.1073/pnas.2025188118">An ecophysiological explanation for manganese enrichment in rock varnish.</a> Proceedings of the National Academy of Sciences, 118(25), p.e2025188118. </p>
							<p id="18">18: He, X., <strong><u>Chadwick, G. L.</u></strong>, Kempes, C. P., Orphan, V. J., & Meile, C. (2021). <a href="https://doi.org/10.1128/mBio.03620-20">Controls on interspecies electron transport and size limitation of anaerobically methane-oxidizing microbial consortia.</a> Mbio, 12(3), e03620-20. </p>
							<p id="17">17: He, X., <strong><u>Chadwick, G. L.</u></strong>, Jiménez Otero, F., Orphan, V., & Meile, C. (2021). <a href="https://doi.org/10.1002/celc.202100111">Spatially Resolved Electron Transport through Anode-Respiring Geobacter sulfurreducens Biofilms: Controls and Constraints.</a> ChemElectroChem, 8(10), 1747-1758. </p>
							<p id="16">16: Mickol, R.L., Louyakis, A.S., Kee, H.L., Johnson, L.K., Dawson, S.C., Hargreaves, K.R., <strong><u>Chadwick, G. L.</u></strong>, Newman, D.K., Leadbetter, J.R. and Brown, C.T., (2021). <a href="https://doi.org/10.1128/MRA.01302-20">Draft Genome Sequence of the Free-Living, Iridescent Bacterium Tenacibaculum mesophilum Strain ECR.</a> Microbiology resource announcements, 10(1), pp.e01302-20. </p>
							<p id="15">15: <strong><u>Chadwick, G. L.</u></strong>*, Otero, F. J.*, Gralnick, J. A., Bond, D. R., & Orphan, V. J. (2019). <a href="https://doi.org/10.1073/pnas.1912498116">NanoSIMS imaging reveals metabolic stratification within current-producing biofilms.</a> PNAS, 201912498. *Authors contributed equally to this work. </p>
							<p id="14">14: Bublitz, D. C., <strong><u>Chadwick, G. L.</u></strong>, Magyar, J. S., Sandoz, K. M., Brooks, D., Mesnage, S., Ladinsky, M. S., Garber, A. I., Bjorkman, P. J., Orphan, V. J., McCutcheon, J. P. (2019). <a href="https://doi.org/10.1016/j.cell.2019.08.054">Peptidoglycan production by an insect-bacterial mosaic.</a> Cell, 179:j.cell.2019.08.054. </p>
							<p id="13">13: He, X., <strong><u>Chadwick, G. L.</u></strong>, Kempes, C., Shi, Y., McGlynn, S., Orphan, V., & Meile, C. (2019). <a href="https://doi.org/10.1111/1462-2920.14507">Microbial interactions in the anaerobic oxidation of methane: model simulations constrained by process rates and activity patterns.</a> Environmental microbiology, 21(2), 631-647. </p>
							<p id="12">12: Boyd, J.A., Jungbluth, S.P., Leu, A.O., Evans, P.N., Woodcroft, B.J., <strong><u>Chadwick, G. L.</u></strong>, Orphan, V.J., Amend, J.P., Rappé, M.S. and Tyson, G.W. (2019). <a href="https://www.nature.com/articles/s41396-018-0343-2">Divergent methyl-coenzyme M reductase genes in a deep-subseafloor Archaeoglobi.</a> The ISME journal, 13(5), 1269. </p>
							<p id="11">11: Bird, L. R., Dawson, K. S., <strong><u>Chadwick, G. L.</u></strong>, Fulton, J. M., Orphan, V. J., & Freeman, K. H. (2019). <a href="https://doi.org/10.1111/gbi.12354">Carbon isotopic heterogeneity of coenzyme F430 and membrane lipids in methane-oxidizing archaea.</a> Geobiology. </p>
							<p id="10">10: <strong><u>Chadwick, G. L.</u></strong>, Hemp, J., Fischer, W. W., & Orphan, V. J. (2018). <a href="https://www.nature.com/articles/s41396-018-0210-1">Convergent evolution of unusual complex I homologs with increased proton pumping capacity: energetic and ecological implications.</a> The ISME journal, 12(11), 2668. </p>
							<p id="9">9: Dekas, A.E., Fike, D.A., <strong><u>Chadwick, G. L.</u></strong>, Green-Saxena, A., Fortney, J., Connon, S.A., Dawson, K.S. and Orphan, V.J. (2018). <a href="https://doi.org/10.1111/1462-2920.14342">Widespread nitrogen fixation in sediments from diverse deep-sea sites of elevated carbon loading.</a> Environmental microbiology, 20(12), 4281-4296. </p>
							<p id="8">8: McGlynn, S.E., <strong><u>Chadwick, G. L.</u></strong>, O'Neill, A., Mackey, M., Thor, A., Deerinck, T.J., Ellisman, M.H. and Orphan, V.J. (2018). <a href="https://doi.org/10.1128/AEM.00399-18">Subgroup characteristics of marine methane-oxidizing ANME-2 archaea and their syntrophic partners as revealed by integrated multimodal analytical microscopy.</a> Appl. Environ. Microbiol., 84(11), e00399-18. </p>
							<p id="7">7: Scheller, S., Yu, H., <strong><u>Chadwick, G. L.</u></strong>, McGlynn, S. E., & Orphan, V. J. (2016). <a href="https://doi.org/10.1126/science.aad7154">Artificial electron acceptors decouple archaeal methane oxidation from sulfate reduction.</a> Science, 351(6274), 703-707. </p>
							<p id="6">6: Dekas, A. E., Connon, S. A., <strong><u>Chadwick, G. L.</u></strong>, Trembath-Reichert, E., & Orphan, V. J. (2016). <a href="https://doi.org/10.1038/ismej.2015.145">Activity and interactions of methane seep microorganisms assessed by parallel transcription and FISH-NanoSIMS analyses.</a> The ISME journal, 10(3), 678-692. </p>
							<p id="5">5: McGlynn, S. E.*, <strong><u>Chadwick, G. L.</u></strong>*, Kempes, C. P., & Orphan, V. J. (2015). <a href="https://www.nature.com/articles/nature15512">Single cell activity reveals direct electron transfer in methanotrophic consortia.</a> Nature, 526(7574), 531-535. *Authors contributed equally to this work. </p>
							<p id="4">4: Evans, P. N., Parks, D. H., <strong><u>Chadwick, G. L.</u></strong>, Robbins, S. J., Orphan, V. J., Golding, S. D., & Tyson, G. W. (2015). <a href="https://www.science.org/doi/full/10.1126/science.aac7745">Methane metabolism in the archaeal phylum Bathyarchaeota revealed by genome-centric metagenomics.</a> Science, 350(6259), 434-438. </p>
							<p id="3">3: Tavormina, P. L., Hatzenpichler, R., McGlynn, S., <strong><u>Chadwick, G. L.</u></strong>, Dawson, K. S., Connon, S. A., & Orphan, V. J. (2015). <a href="https://doi.org/10.1099/ijs.0.062927-0">Methyloprofundus sedimenti gen. nov., sp. nov., an obligate methanotroph from ocean sediment belonging to the ‘deep sea-1’ clade of marine methanotrophs.</a> International journal of systematic and evolutionary microbiology, 65(1), 251-259. </p>
							<p id="2">2: Dekas, A. E., <strong><u>Chadwick, G. L.</u></strong>, Bowles, M. W., Joye, S. B., & Orphan, V. J. (2014). <a href="https://doi.org/10.1111/1462-2920.12247">Spatial distribution of nitrogen fixation in methane seep sediment and the role of the ANME archaea.</a> Environmental microbiology, 16(10), 3012-3029. </p>
							<p id="1">1: Levin, L. A., Orphan, V. J., Rouse, G. W., Rathburn, A. E., Ussler III, W., Cook, G. S., ... & Strickrott, B. (2012). <a href="https://doi.org/10.1098/rspb.2012.0205">A hydrothermal seep on the Costa Rica margin: middle ground in a continuum of reducing ecosystems.</a> Proceedings of the Royal Society B: Biological Sciences, 279(1738), 2580-2588. </p>
					</section>
				<!-- Six -->
					<section id="seven">
						<h2>Presentations</h2>
						<p>
						<strong><u>Chadwick, G. L.</u></strong> (2022). Using nanoSIMS to answer interesting spatial questions. (Talk) International Geobiology Course 2022. Mammoth Lakes, CA
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong> (2021). Genomic determinants of reverse methanogenesis in archaea. (Talk) Integrative Genomics Institute Seminar Series. Virtual.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong> (2021). Physiological and transcriptional response to the repression of methanogenesis in a methanogen. (Talk) West Coast Bacterial Physiologists. Virtual.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong> (2020). Genomic determinants of reverse methanogenesis in archaea. (Talk) Molecular and Cell Biology Department - Genetics, Genomics & Development retreat. Virtual.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong> (2020). How to anaerobically oxidize hydrocarbons: lessons from hydrothermal (and cold) marine sediments. (Invited Talk) Thermal Biology Institute Seminar Series, MSU. Virtual.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong> (2020). Cell to Ecosystem: Understanding methane and associated nutrient cycling by sediment-hosted syntrophic consortia and their viral predators. (Invited Talk) DOE-BER 2020 Genomic Sciences Program Annual PI Meeting. Arlington, Virginia.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong> (2019). Ultrafast swimming in a novel genus of purple sulfur bacteria.  (Talk) West Coast Bacterial Physiologists. Pacific Palisades, California
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong> (2019). Methanotrophy: looking backward. (Invited Talk) COST Action CA17120 Chemobrionics 30/80 Meeting. Granada, Spain.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong>, Jiménez-Otero, F., Deerinck, T., He, X., Skennerton, C. T., Meile, C., Ellisman, M., Bond, D. R., Orphan, V. J. (2018). Implications of spatial structure in methanotrophic ANME-SRB consortia. (Poster) Molecular Basis of Microbial One-Carbon Metabolism, Gordon Research Conference.  Sunday River, Maine.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong>, Hemp, J., Fischer, W. W., & Orphan, V. J. (2017). Predicting the function of novel of complex I homologs from poorly studied and uncultured organisms: hints from primary sequences, crystal structures and phylogenies. (Poster) NeLLi 2017: From New Lineages of Life To New Functions. Walnut Creek, California.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong>, Jiménez-Otero, F., Gralnick, J. A., Bond, D. R., Orphan, V. J. (2017). Geobacter activity in electrode biofilms is spatially dependent. (Poster) ISMET6 International Society for Microbial Electrochemistry and Technology Meeting. Lisbon, Portugal.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong>, Hemp, J., Fischer, W. W., & Orphan, V. J. (2017). Convergent evolution of respiratory complex I homologs with additional proton pumping capacity. (Seminar series talk) MicroMorning, Caltech.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong> and Orphan, V. J. (2015). Genomic insights into the biochemistry and bioenergetics of anaerobic methanotrophic archaea. (Poster) Archaea: Ecology, Metabolism & Molecular Biology, Gordon Research Conference.  Sunday River, Maine.
						<br><br>
						<strong><u>Chadwick, G. L.</u></strong>, Dekas, A. E., Connon, S. A., and Orphan, V. J. (2011). Investigations into ammonium dependent shut off of nitrogen fixation in methane seep sediments from Hydrate Ridge North. (Poster) Southern California Geobiology Symposium, USC.
						</p>
					</section>


				<!-- Five -->
				<!--
					<section id="four">
						<h2>Elements</h2>

						<section>
							<h4>Text</h4>
							<p>This is <b>bold</b> and this is <strong>strong</strong>. This is <i>italic</i> and this is <em>emphasized</em>.
							This is <sup>superscript</sup> text and this is <sub>subscript</sub> text.
							This is <u>underlined</u> and this is code: <code>for (;;) { ... }</code>. Finally, <a href="#">this is a link</a>.</p>
							<hr />
							<header>
								<h4>Heading with a Subtitle</h4>
								<p>Lorem ipsum dolor sit amet nullam id egestas urna aliquam</p>
							</header>
							<p>Nunc lacinia ante nunc ac lobortis. Interdum adipiscing gravida odio porttitor sem non mi integer non faucibus ornare mi ut ante amet placerat aliquet. Volutpat eu sed ante lacinia sapien lorem accumsan varius montes viverra nibh in adipiscing blandit tempus accumsan.</p>
							<header>
								<h5>Heading with a Subtitle</h5>
								<p>Lorem ipsum dolor sit amet nullam id egestas urna aliquam</p>
							</header>
							<p>Nunc lacinia ante nunc ac lobortis. Interdum adipiscing gravida odio porttitor sem non mi integer non faucibus ornare mi ut ante amet placerat aliquet. Volutpat eu sed ante lacinia sapien lorem accumsan varius montes viverra nibh in adipiscing blandit tempus accumsan.</p>
							<hr />
							<h2>Heading Level 2</h2>
							<h3>Heading Level 3</h3>
							<h4>Heading Level 4</h4>
							<h5>Heading Level 5</h5>
							<h6>Heading Level 6</h6>
							<hr />
							<h5>Blockquote</h5>
							<blockquote>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan faucibus. Vestibulum ante ipsum primis in faucibus lorem ipsum dolor sit amet nullam adipiscing eu felis.</blockquote>
							<h5>Preformatted</h5>
							<pre><code>i = 0;

while (!deck.isInOrder()) {
print 'Iteration ' + i;
deck.shuffle();
i++;
}

print 'It took ' + i + ' iterations to sort the deck.';</code></pre>
						</section>

						<section>
							<h4>Lists</h4>
							<div class="row">
								<div class="col-6 col-12-xsmall">
									<h5>Unordered</h5>
									<ul>
										<li>Dolor pulvinar etiam magna etiam.</li>
										<li>Sagittis adipiscing lorem eleifend.</li>
										<li>Felis enim feugiat dolore viverra.</li>
									</ul>
									<h5>Alternate</h5>
									<ul class="alt">
										<li>Dolor pulvinar etiam magna etiam.</li>
										<li>Sagittis adipiscing lorem eleifend.</li>
										<li>Felis enim feugiat dolore viverra.</li>
									</ul>
								</div>
								<div class="col-6 col-12-xsmall">
									<h5>Ordered</h5>
									<ol>
										<li>Dolor pulvinar etiam magna etiam.</li>
										<li>Etiam vel felis at lorem sed viverra.</li>
										<li>Felis enim feugiat dolore viverra.</li>
										<li>Dolor pulvinar etiam magna etiam.</li>
										<li>Etiam vel felis at lorem sed viverra.</li>
										<li>Felis enim feugiat dolore viverra.</li>
									</ol>
									<h5>Icons</h5>
									<ul class="icons">
										<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
										<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
										<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
										<li><a href="#" class="icon brands fa-github"><span class="label">Github</span></a></li>
										<li><a href="#" class="icon brands fa-dribbble"><span class="label">Dribbble</span></a></li>
										<li><a href="#" class="icon brands fa-tumblr"><span class="label">Tumblr</span></a></li>
									</ul>
								</div>
							</div>
							<h5>Actions</h5>
							<ul class="actions">
								<li><a href="#" class="button primary">Default</a></li>
								<li><a href="#" class="button">Default</a></li>
							</ul>
							<ul class="actions small">
								<li><a href="#" class="button primary small">Small</a></li>
								<li><a href="#" class="button small">Small</a></li>
							</ul>
							<div class="row">
								<div class="col-6 col-12-small">
									<ul class="actions stacked">
										<li><a href="#" class="button primary">Default</a></li>
										<li><a href="#" class="button">Default</a></li>
									</ul>
								</div>
								<div class="col-6 col-12-small">
									<ul class="actions stacked">
										<li><a href="#" class="button primary small">Small</a></li>
										<li><a href="#" class="button small">Small</a></li>
									</ul>
								</div>
								<div class="col-6 col-12-small">
									<ul class="actions stacked">
										<li><a href="#" class="button primary fit">Default</a></li>
										<li><a href="#" class="button fit">Default</a></li>
									</ul>
								</div>
								<div class="col-6 col-12-small">
									<ul class="actions stacked">
										<li><a href="#" class="button primary small fit">Small</a></li>
										<li><a href="#" class="button small fit">Small</a></li>
									</ul>
								</div>
							</div>
						</section>

						<section>
							<h4>Table</h4>
							<h5>Default</h5>
							<div class="table-wrapper">
								<table>
									<thead>
										<tr>
											<th>Name</th>
											<th>Description</th>
											<th>Price</th>
										</tr>
									</thead>
									<tbody>
										<tr>
											<td>Item One</td>
											<td>Ante turpis integer aliquet porttitor.</td>
											<td>29.99</td>
										</tr>
										<tr>
											<td>Item Two</td>
											<td>Vis ac commodo adipiscing arcu aliquet.</td>
											<td>19.99</td>
										</tr>
										<tr>
											<td>Item Three</td>
											<td> Morbi faucibus arcu accumsan lorem.</td>
											<td>29.99</td>
										</tr>
										<tr>
											<td>Item Four</td>
											<td>Vitae integer tempus condimentum.</td>
											<td>19.99</td>
										</tr>
										<tr>
											<td>Item Five</td>
											<td>Ante turpis integer aliquet porttitor.</td>
											<td>29.99</td>
										</tr>
									</tbody>
									<tfoot>
										<tr>
											<td colspan="2"></td>
											<td>100.00</td>
										</tr>
									</tfoot>
								</table>
							</div>

							<h5>Alternate</h5>
							<div class="table-wrapper">
								<table class="alt">
									<thead>
										<tr>
											<th>Name</th>
											<th>Description</th>
											<th>Price</th>
										</tr>
									</thead>
									<tbody>
										<tr>
											<td>Item One</td>
											<td>Ante turpis integer aliquet porttitor.</td>
											<td>29.99</td>
										</tr>
										<tr>
											<td>Item Two</td>
											<td>Vis ac commodo adipiscing arcu aliquet.</td>
											<td>19.99</td>
										</tr>
										<tr>
											<td>Item Three</td>
											<td> Morbi faucibus arcu accumsan lorem.</td>
											<td>29.99</td>
										</tr>
										<tr>
											<td>Item Four</td>
											<td>Vitae integer tempus condimentum.</td>
											<td>19.99</td>
										</tr>
										<tr>
											<td>Item Five</td>
											<td>Ante turpis integer aliquet porttitor.</td>
											<td>29.99</td>
										</tr>
									</tbody>
									<tfoot>
										<tr>
											<td colspan="2"></td>
											<td>100.00</td>
										</tr>
									</tfoot>
								</table>
							</div>
						</section>

						<section>
							<h4>Buttons</h4>
							<ul class="actions">
								<li><a href="#" class="button primary">Primary</a></li>
								<li><a href="#" class="button">Default</a></li>
							</ul>
							<ul class="actions">
								<li><a href="#" class="button large">Large</a></li>
								<li><a href="#" class="button">Default</a></li>
								<li><a href="#" class="button small">Small</a></li>
							</ul>
							<ul class="actions fit">
								<li><a href="#" class="button primary fit">Fit</a></li>
								<li><a href="#" class="button fit">Fit</a></li>
							</ul>
							<ul class="actions fit small">
								<li><a href="#" class="button primary fit small">Fit + Small</a></li>
								<li><a href="#" class="button fit small">Fit + Small</a></li>
							</ul>
							<ul class="actions">
								<li><a href="#" class="button primary icon solid fa-download">Icon</a></li>
								<li><a href="#" class="button icon solid fa-download">Icon</a></li>
							</ul>
							<ul class="actions">
								<li><span class="button primary disabled">Primary</span></li>
								<li><span class="button disabled">Default</span></li>
							</ul>
						</section>

						<section>
							<h4>Form</h4>
							<form method="post" action="#">
								<div class="row gtr-uniform gtr-50">
									<div class="col-6 col-12-xsmall">
										<input type="text" name="demo-name" id="demo-name" value="" placeholder="Name" />
									</div>
									<div class="col-6 col-12-xsmall">
										<input type="email" name="demo-email" id="demo-email" value="" placeholder="Email" />
									</div>
									<div class="col-12">
										<select name="demo-category" id="demo-category">
											<option value="">- Category -</option>
											<option value="1">Manufacturing</option>
											<option value="1">Shipping</option>
											<option value="1">Administration</option>
											<option value="1">Human Resources</option>
										</select>
									</div>
									<div class="col-4 col-12-small">
										<input type="radio" id="demo-priority-low" name="demo-priority" checked>
										<label for="demo-priority-low">Low Priority</label>
									</div>
									<div class="col-4 col-12-small">
										<input type="radio" id="demo-priority-normal" name="demo-priority">
										<label for="demo-priority-normal">Normal Priority</label>
									</div>
									<div class="col-4 col-12-small">
										<input type="radio" id="demo-priority-high" name="demo-priority">
										<label for="demo-priority-high">High Priority</label>
									</div>
									<div class="col-6 col-12-small">
										<input type="checkbox" id="demo-copy" name="demo-copy">
										<label for="demo-copy">Email me a copy of this message</label>
									</div>
									<div class="col-6 col-12-small">
										<input type="checkbox" id="demo-human" name="demo-human" checked>
										<label for="demo-human">I am a human and not a robot</label>
									</div>
									<div class="col-12">
										<textarea name="demo-message" id="demo-message" placeholder="Enter your message" rows="6"></textarea>
									</div>
									<div class="col-12">
										<ul class="actions">
											<li><input type="submit" value="Send Message" class="primary" /></li>
											<li><input type="reset" value="Reset" /></li>
										</ul>
									</div>
								</div>
							</form>
						</section>

						<section>
							<h4>Image</h4>
							<h5>Fit</h5>
							<div class="box alt">
								<div class="row gtr-50 gtr-uniform">
									<div class="col-12"><span class="image fit"><img src="images/fulls/05.jpg" alt="" /></span></div>
									<div class="col-4"><span class="image fit"><img src="images/thumbs/01.jpg" alt="" /></span></div>
									<div class="col-4"><span class="image fit"><img src="images/thumbs/02.jpg" alt="" /></span></div>
									<div class="col-4"><span class="image fit"><img src="images/thumbs/03.jpg" alt="" /></span></div>
									<div class="col-4"><span class="image fit"><img src="images/thumbs/04.jpg" alt="" /></span></div>
									<div class="col-4"><span class="image fit"><img src="images/thumbs/05.jpg" alt="" /></span></div>
									<div class="col-4"><span class="image fit"><img src="images/thumbs/06.jpg" alt="" /></span></div>
									<div class="col-4"><span class="image fit"><img src="images/thumbs/03.jpg" alt="" /></span></div>
									<div class="col-4"><span class="image fit"><img src="images/thumbs/02.jpg" alt="" /></span></div>
									<div class="col-4"><span class="image fit"><img src="images/thumbs/01.jpg" alt="" /></span></div>
								</div>
							</div>
							<h5>Left &amp; Right</h5>
							<p><span class="image left"><img src="images/avatar.jpg" alt="" /></span>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent.</p>
							<p><span class="image right"><img src="images/avatar.jpg" alt="" /></span>Fringilla nisl. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent. Donec accumsan interdum nisi, quis tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent tincidunt felis sagittis eget. tempus euismod. Vestibulum ante ipsum primis in faucibus vestibulum. Blandit adipiscing eu felis iaculis volutpat ac adipiscing accumsan eu faucibus. Integer ac pellentesque praesent.</p>
						</section>

					</section>
				-->

			</div>

		<!-- Footer -->
			<footer id="footer">
				<div class="inner">
					<ul class="icons">
						<li><a href="https://www.linkedin.com/in/grayson-chadwick-17a629243/" class="icon brands fa-linkedin"><span class="label">LinkedIn</span></a></li>
						<li><a href="https://github.com/gchadwick" class="icon brands fa-github"><span class="label">Github</span></a></li>
						<li><a href="mailto:graysemail@gmail.com" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
					</ul>
					<ul class="copyright">
						<li>Portrait by Jared R. Leadbetter. <br>All other photos and figures generated by me.<br>Design modified from Strata template: <a href="http://html5up.net">HTML5 UP</a></li>
					</ul>
				</div>
			</footer>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.poptrox.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>